edx_proctoring/templates/proctored_exam/error.html

```diff
@@ -8,11 +8,9 @@ <h3>
 
   <p>
     {% blocktrans %}
-      A technical error has occurred with your proctored exam. All
-      exam data, including completed answers, has been lost. To
-      resolve this error, please
-      contact <a href="mailto:{{tech_support_email}}">technical
-      support</a>.
+      A technical error has occurred with your proctored exam. Please
+      reach out to <a href="{{link_urls.contact_us}}" target="_blank">
```

Comment:
https://github.com/edx/edx-proctoring/blob/master/edx_proctoring/templates/proctored_exam/ready_to_start.html#L28  is the solution I copied to here.